guides/doc-Provisioning_Guide/topics/Networking.adoc

```diff
@@ -3,31 +3,50 @@
 
-New hosts must have access to your {SmartProxyServer}. A {SmartProxyServer} can be either {ProjectServer}'s integrated {SmartProxy} or an external {SmartProxyServer}. You might want to provision hosts from an external {SmartProxyServer} when the hosts are on isolated networks and cannot connect to the {ProjectServer} directly, or when the content is synchronized with the {SmartProxyServer}. Provisioning using the external {SmartProxyServer} can save on network bandwidth.
+New hosts must have access to your {SmartProxyServer}. {SmartProxyServer} can be either {ProjectServer}'s integrated {SmartProxy} or an external {SmartProxyServer}. You might want to provision hosts from an external {SmartProxyServer} when the hosts are on isolated networks and cannot connect to the {ProjectServer} directly, or when the content is synchronized with the {SmartProxyServer}. Provisioning using the external {SmartProxyServer} can save on network bandwidth.
 
-Configuring the {SmartProxyServer} has two basic requirements:
+Configuring {SmartProxyServer} has two basic requirements:
 
 . Configuring network services. This includes:
   * Content delivery services
   * Network services (DHCP, DNS, and TFTP)
   * Puppet configuration
 . Defining network resource data in {ProjectServer} to help configure network interfaces on new hosts.
 
-The following instructions have similar applications to configuring standalone {SmartProxyServer}s managing a specific network. To configure {Project} to use external DHCP, DNS, and TFTP services, see link:{BaseURL}installing_satellite_server_from_a_connected_network/configuring_external_services[Configuring External Services] in _Installing {ProjectServer} from a Connected Network_.
+The following instructions have similar applications to configuring standalone {SmartProxies} managing a specific network. To configure {Project} to use external DHCP, DNS, and TFTP services, see link:{BaseURL}installing_satellite_server_from_a_connected_network/configuring_external_services[Configuring External Services] in _Installing {ProjectServer} from a Connected Network_.
 
-.DHCP Ranges
-You can define the same DHCP range in {ProjectServer} for both Discovered and Provisioned systems, but use a separate range for each service within the same subnet.
+.Network Resources
+
+{Project} contains networking resources that you must set up and configure to create a host. {Project} includes the following networking resources:
+
+Domain::
+You must assign every host that is managed by {Project} to a domain. Using the domain, {Project} can manage A, AAAA and PTR records. If there is no record integration, you still must create and associate at least one domain. Domains are included in the naming conventions {Project} hosts, for example, a host `test123` in the `example.com` domain has the host name `test123.example.com`.
```

Comment:
"If you don't want Satellite to manage your DNS servers, you still must create..." in some better form but your proposal reads weird to me.